docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -235,82 +242,411 @@ $ tsh login --proxy=mytenant.teleport.sh
   SSO users should configure multi-factor authentication in their SSO provider.
 </Admonition>
 
-## U2F
+## Configure per-session MFA
 
-WebAuthn has replaced U2F in Teleport from previous versions. If you haven't configured U2F
-before, no further action is necessary—any U2F devices are automatically
-supported.
+Ensure that multi-factor authentication is required for each session, not just at initial login, 
+to maintain continuous security. Teleport's per-session MFA enhances security by protecting 
+against compromised on-disk certificates. It requires additional MFA checks when initiating new SSH, Kubernetes, database, or desktop sessions.
 
-If you have an existing U2F configuration, but haven't explicitly configured
-WebAuthn yet, Teleport will automatically derive your WebAuthn configuration
-from your existing U2F configuration.
+Teleport supports requiring additional multi-factor authentication checks
+when starting new:
 
-You may write the WebAuthn configuration yourself, but keep the U2F `app_id`
-field. Doing so ensures that any already-registered U2F devices won't need to be
-re-registered.
+- SSH connections (a single `tsh ssh` call, Web UI SSH session or Teleport Connect SSH session)
+- Kubernetes sessions (a single `kubectl` call)
+- Database sessions (a single `tsh db connect` call)
+- Application sessions
+- Desktop sessions
 
-For example, consider the U2F configuration below:
+<Notice type="note">
+  In addition to per-session MFA, enable login MFA in your SSO provider and/or
+  for all [local Teleport
+  users](../../reference/authentication.mdx#local-no-authentication-connector)
+  to improve security.
+</Notice>
+
+## Step 1/1. Enforce MFA checks
+
+<Tabs>
```

Comment:
Nested tabs don't render very well in our docs engine. I vote we don't use tabs at all here and just show how to use a dynamic resource for all users.